htdocs/js/jquery.dynamictable.js

```diff
@@ -130,6 +149,21 @@
       '<span class="glyphicon glyphicon-chevron-right"></span></a>'
     );
   };
+  var headerAlign = function(table, headers) {
+    var tableHeaders = $(table).find("thead").children().children();
+    var fixedHeaders = $(headers).find("thead").children().children();
```

Comment:
Suggest using this `$(headers).find("thead").children().children().not('.static-col');` to eliminate the need for the `if` in the loop